index.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Climate Fixing Conference </title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/4f02d055e5.js"
+      crossorigin="anonymous"
+    ></script>
+    <script src="https://kit.fontawesome.com/870e4168d3.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css" />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <script src="script.js" defer></script>
+  </head>
+
+  <body>
+    <header>
+      <div class="top-bar">
+        <ul class="top-bar-ul">
+          <li><i class="fab fa-facebook-f"></i></li>
+          <li><i class="fab fa-twitter"></i></li>
+          <li>English</li>
+          <li>Mypage</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+      <div class="header-bar">
+        <div class="logo">
+          <a href="./index.html"
+            ><div class="logo-climate">
+              <i class="fa-duotone fa-hand-holding-seedling"></i>
+            <p>SAVE THE PLANET </p>
+            <!-- <p>IG</p> -->
+          </div></a
+          >
+          <div class="close-btn"><i class="fas fa-times fa-2x"></i></div>
+        </div>
+        <nav>
+          <ul class="nav-ul">
+            <li><a class="menu-link" href="./about.html">About</a></li>
+            <li><a class="menu-link" href="#program">Program</a></li>
+            <li><a class="menu-link" href="#">Join</a></li>
+            <li><a class="menu-link" href="#partner">Sponsors</a></li>
+            <li><a class="menu-link" href="#">News</a></li>
+          </ul>
+        </nav>
+        <button class="campaign">Campaign</button>
+      </div>
+    </header>
+    <main>
+      <section class="intro">
+        <div class="menu">
+          <i class="fas fa-bars fa-2x"></i>
+        </div>
+        <h2>"LET U ALL HELP SAVE OUR CLIMATE"</h2>
```

Comment:
- [ ] (Optional) You have a typo here, you can accept my suggestion and it will automatically correct it. 

```suggestion
        <h2>"LET US ALL HELP SAVE OUR CLIMATE"</h2>
```